### PR TITLE
🥧  more comprehensive bootstrap-master example 🥧

### PR DESCRIPTION
**Pull request by eformat**
add a bootstrap-master chart for the master-child argocd demoing
fixup description and instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,11 +152,19 @@ argocd app sync catz
...
+We can create a master ArgoCD instance in the cluster that can bootstrap other "child" ArgoCD instance(s) for any given project team. This is a good approach if you want each project team to own and operate their own software development tools (jenkins, sonar, argocd, etc) but restrict any elevated permissions they may need e.g.creating argocd Custom Resources Definitions (`CRD's`) or limiting project creation.
 
-1. Login to your ArgoCD master and run to create a new project to manage deployments in the Lab's namespace along with the repositories to be allowed pull from:
+1. Deploy a master instance of argocd if you do not already have one. This is deployed into the `master-argocd` project.
+```
+helm template --dependency-update -f bootstrap-master/values-bootstrap.yaml bootstrap-master | oc apply -f-
+```
```
Comment (springdo) [inline README.md:160]: should this just be pulled straight from the helm-chart repo?

Comment (springdo) [reply README.md:160]: like as in this or something `helm template -f charts/argocd-operator/values.yaml --set namespace=ds-argocd --set name=master-argocd --set instancelabel=rht-labs.com/argocd-master charts/argocd-operator | oc apply -f-`

File: bootstrap-master/values-bootstrap.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# bootstrap a master argocd instance
+bootstrap-project:
+  enabled: true
+  ci_cd_namespace: &ci_cd "master-argocd"
```
Comment (springdo) [inline bootstrap-master/values-bootstrap.yaml:4]: ahhh right, i see what you're doing now.

Comment (eformat) [reply bootstrap-master/values-bootstrap.yaml:4]: err, i think this is the right place for it .. the bootstrap and bootstrap-master charts are chart-of-charts .. the work is being done by the chart dependencies ...

yeah, you would struggle to use the bootstrap chart and cli overrides for this use case.

Comment (springdo) [reply bootstrap-master/values-bootstrap.yaml:4]: Yeah I was more initially thinking about the master deploy - perhaps it's just helm deployed or something but yeah, this works too. And ovs this approach is GitOpsy and pluggable

**Pull request merged**